shell/browser/api/atom_api_system_preferences_mac.mm

```diff
@@ -600,11 +585,21 @@ AVMediaType ParseMediaType(const std::string& media_type) {
     gin_helper::Arguments* args) {
   if (auto type = ParseMediaType(media_type)) {
     if (@available(macOS 10.14, *)) {
-      return ConvertAuthorizationStatus(
-          [AVCaptureDevice authorizationStatusForMediaType:type]);
+      switch ([AVCaptureDevice authorizationStatusForMediaType:type]) {
```

Comment:
usage of `AVAuthorizationStatus` has to be in `@available(macOS 10.14, *)` block